### PR TITLE
Adding graph Makefile

### PR DESCRIPTION
**Pull request by dungwinux**

### CODE REVIEW

File: graph/cpp/Makefile [hunk 1]
```diff
@@ -1,8 +1,9 @@
-.PHONY: all clean
+-include ../../Makefile.defaults
 
-all:
-	./make.sh
+compile:
+	CXX=$(CXX) CXXFLAGS=$(CXXFLAGS) cmake -B build --fresh -DGRAPH_RECURSIVE=ON -DGRAPH_ABSL_FLAT=ON
```
Comment (nicovank) [inline graph/cpp/Makefile]:
```suggestion
	cmake -B build --fresh -DCMAKE_BUILD_TYPE=Release -DGRAPH_RECURSIVE=ON -DGRAPH_ABSL_FLAT=ON
```

`CXX` and `CXXFLAGS` should already be passed down from environment. We should specify a release build type.

File: graph/cpp/graph.cpp [hunk 2]
```diff
@@ -469,7 +469,7 @@ int main(int argc, char *argv[]) {
 	printf("[config] -O3\n");
 #else
 	printf("[config] badness.\n");
-	return 1;
+	// return 1;
```
Comment (nicovank) [inline graph/cpp/graph.cpp]: Let's just get rid of this whole `#if` block.

**Review by nicovank — APPROVED** [review]
I assume that this requires setting the environment variables `JAVAC`, `GO`, etc when running those?

File: graph/cpp/graph.cpp (hunk 2 above)
Comment (dungwinux) [reply graph/cpp/graph.cpp]: Fixed [87e506c](https://github.com/nicovank/LangBench/pull/4/commits/87e506ce98c3f5e2cc3a250f722da6da76ef4904)

**Comment by dungwinux** [comment]
> I assume that this requires setting the environment variables `JAVAC`, `GO`, etc when running those?

Yes. It is the same for Energy-Languages, no?

**Comment by nicovank** [comment]
Technically we have a couple `ifndef` to set these up to defaults but this is fine.

**Pull request merged**